requirements.py_docs.txt

```diff
@@ -0,0 +1,3 @@
+docutils==0.17.1
```

Comment:
Nit: use `...py-docs.txt`, for consistency with tox env name; same for `sphinix-docs`